webtesting/src/App.js

```diff
@@ -3,23 +3,46 @@ import Display from './components/Display';
 import Dashboard from './components/Dashboard';
 import './App.css';
 
-function App() {
+const App = () => {
+
+  const [strikes, setStrikes] = useState(0);
+  const [balls, setBalls] = useState(0);
+
+  const addStrike = () => {
```

Comment:
add comments to the logic as well 
like this function will add up until 3 and then reset to 0